{"name": "test_get_all_user_info[True-0-查询成功]", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 401 】, 返回信息：请求访问：/users，认证失败，无法访问系统资源 \nassert False == True\n +  where False = <core.result_base.ResultBase object at 0x108fdaeb0>.success", "trace": "self = <testcases.api_test.test_01_get_user_info.TestGetUserInfo object at 0x108f80460>, except_result = True, except_code = 0, except_msg = '查询成功'\n\n    @allure.story(\"用例--获取全部用户信息\")\n    @allure.description(\"该用例是针对获取所有用户信息接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"except_result, except_code, except_msg\",\n                             api_data[\"test_get_all_user_info\"])\n    def test_get_all_user_info(self, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        step_1()\n        result = get_all_user_info()\n        # print(result.__dict__)\n        assert result.response.status_code == 200\n>       assert result.success == except_result, result.error\nE       AssertionError: 接口返回码是 【 401 】, 返回信息：请求访问：/users，认证失败，无法访问系统资源 \nE       assert False == True\nE        +  where False = <core.result_base.ResultBase object at 0x108fdaeb0>.success\n\ntestcases/api_test/test_01_get_user_info.py:37: AssertionError"}, "description": "该用例是针对获取所有用户信息接口的测试", "steps": [{"name": "步骤1 ==>> 获取所有用户信息", "status": "passed", "start": 1735623705535, "stop": 1735623705535}], "attachments": [{"name": "log", "source": "3496f3b7-8e06-4ef3-bf94-f38b844f8908-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "b6a35b14-042d-4e01-8c29-5f7fe8d26772-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'查询成功'"}], "start": 1735623705534, "stop": 1735623706583, "uuid": "10bb498b-0576-485e-b92a-00e2371d32e5", "historyId": "8fa120af048dc3f86b08ef0469022873", "testCaseId": "c8c195e0d2fed438fa6bca8fa8984833", "fullName": "testcases.api_test.test_01_get_user_info.TestGetUserInfo#test_get_all_user_info", "labels": [{"name": "severity", "value": "trivial"}, {"name": "story", "value": "用例--获取全部用户信息"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "feature", "value": "获取用户信息模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_01_get_user_info"}, {"name": "subSuite", "value": "TestGetUserInfo"}, {"name": "host", "value": "zhouqideMBP.lan"}, {"name": "thread", "value": "68918-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_01_get_user_info"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}